group_vars/all

```diff
@@ -53,3 +53,21 @@ ansible_git_repository: git://github.com/buildbot/buildbot-infra
 
 # syslog server to which all syslog messages should be forwarded
 global_syslog_server: 192.168.80.51
+
+# `hosts` declares ip addresses for _all_ machines and jails under Ansible
+# management.
+hosts:
+  ns1:
+    vlan0: "192.168.80.236"
```

Comment:
And maybe these should be keyed by purpose:

    hosts:
        buildbot:
            local: {"interface": "lo0", "ip": "127.0.0.239"}
            public: {"interface": "lagg0", "ip": "140.211.10.239", "hostname": "buildbot.buildbot.net"}
            private: {"interface": "vlan0", "ip": "192.168.80.239", "hostname": "buildbot.int.buildbot.net"}